galasa-parent/dev.galasa.framework.api.ras/src/test/java/dev/galasa/framework/api/ras/RequestorRasTest.java

```diff
@@ -0,0 +1,61 @@
+package dev.galasa.framework.api.ras;
+import java.util.HashMap;
+import java.util.Map;
+import dev.galasa.framework.api.ras.internal.ExtractQuerySort;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+import org.junit.Test;
+
+public class RequestorRasTest {
+	
+	@Test
+	public void testAssertsTrueWhenParameterDoesntExist() {
+		Map<String, String[]> map = new HashMap<>();
+		
+		String[] stringArr = {"requestor:asc"};
+		
+		map.put("sort", stringArr);
+		
+		assertTrue(ExtractQuerySort.isAscending(map, "jim"));
+	}
+	
+	@Test
+	public void testAssertsTrueForRequestorAsc() {
+		
+		Map<String, String[]> map = new HashMap<>();
+		
+		String[] stringArr = {"requestor:asc"};
+		
+		map.put("sort", stringArr);
+		
+		assertTrue(ExtractQuerySort.isAscending(map, "requestor"));
+		
+	}
+	
+	@Test
+	public void testAssertsFalseForRequestorDesc() {
+		
+		Map<String, String[]> map = new HashMap<>();
+		
+		String[] stringArr = {"requestor:desc"};
+		
+		map.put("sort", stringArr);
+		
+		assertFalse(ExtractQuerySort.isAscending(map, "requestor"));
+		
+	}
+	
+	@Test
+	public void testMultipleSortOptions() {
+		
+		Map<String, String[]> map = new HashMap<>();
+		
+		String[] stringArr = {"requestor:asc,testclass:asc,bundle:asc"};
+		
+		map.put("sort", stringArr);
+		
+		assertTrue(ExtractQuerySort.isAscending(map, "requestor"));
```

Comment:
add asserttrues for this test case for all 3 and add an assert false for bob.  and and assert true for jenny which is not included.